pkg/authentication/handlers.go

```diff
@@ -86,8 +140,8 @@ func (a *API) handleLogin(w http.ResponseWriter, r *http.Request) {
 
 	rawIDToken, ok := oauth2Token.Extra("id_token").(string)
 	if !ok {
-		a.logger.Error("unable to retrieve ID token")
-		badRequest(w, errors.New("unable to retrieve ID token"))
+		a.logger.Error(noIDTokenMsg)
+		badRequest(w, noIDTokenErr)
```

Comment:
missed this in the prev pr, but basically (and this is a matter of preference) handler code is quite limited and i do believe there is some benefit in having it as explicit as possible

now having a helper function is great but at the same time makes u have another jump in the codebase to understand what's going on

i'm happy to leave it there but unless it get's used across the board in all the API i would say that in this case just makes it a bit more hideous to understand what's going on compared to any other pkg handlers